.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Github action job to test core java library features on
+# downstream client libraries before they are released.
+on:
+  push:
+    paths:
+      - 'bigtable/**'
+    branches:
+    - main
+  pull_request:
+    paths:
+      - 'bigtable/**'
+name: conformance
+jobs:
+  conformance:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        path: google-cloud-go
+    - uses: actions/checkout@v3
+      with:
+        repository: googleapis/cloud-bigtable-clients-test
+        ref: main
+        path: cloud-bigtable-clients-test
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.20.2'
```

Comment:
Should we run this for multiple golang versions like in other clients?

https://github.com/googleapis/nodejs-bigtable/pull/1349/files#diff-7bfd467904d753bc34e8f91ffbb8165dc048bbe3610af168c600072d0ec473c0R25-R27

Reply:
The testproxy can be run 1.19, 1.20, 1.21 but the conformance tests themselves can be run only on 1.20.2
Modified accordingly.

Reply:
Got it. Thanks for the explanation.